configure

```diff
@@ -13,23 +13,19 @@ inpath()
     return 1
 }
 
-if ( inpath bash ); then
-    pp=""
-    for p in python3 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3 python3.2 python3.1 python2 python2.7 python
-    do
-        if ( inpath $p ); then
-            pp="$p"
-            break
-        fi
-    done
-    if [ "$pp" != "" ]; then
-        exec $pp `dirname $0`/make/configure.py "$@"
-        exit 0
-    else
-        echo "ERROR: no suitable version of python found."
+pp=""
+for p in python3 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3 python3.2 python3.1 python2 python2.7 python
+do
+    if ( inpath $p ); then
+        pp="$p"
```

Comment:
Looks like this commit reversed some changes from the previous.

Reply:
Sorry, but I'm not sure which changes you're referring to. The only changes I can see for `configure` are the one that removes the bash check, and the one that updates the python versions and simplifies the script to match `make/python_launcher`.